contracts/KetlAttestation.sol

```diff
@@ -145,13 +151,13 @@ contract KetlAttestation is
     minimumEntanglementCounts[_id] = _minimumEntanglementCount;
   }
 
-  function setMaxEntanglementsPerAttestation(
-    uint _attestationHash,
-    uint16 _maxEntanglementsPerAttestation
+  function setMaxEntanglementsPerAttestationType(
+    uint _attestationType,
```

Comment:
Again, maybe we can use `enum` here?

Reply:
Same as: https://github.com/BigWhaleLabs/ketl-attestation-token/pull/33#discussion_r1275238450